python/app.py

```diff
@@ -12,6 +12,7 @@
 from database import create_db_command, db
 from page.auth.route import auth_page_bp
 from page.problem.route import problem_page_bp
+from page.submission.route import submission_page_bp
 from page.route import page
```

Comment:
依照 PEP8 的風格，對於每一段 import section 應保持字典序排序，故為

```suggestion
from page.problem.route import problem_page_bp
from page.route import page
from page.submission.route import submission_page_bp
```